lib/HTFeed/Stage/Download.pm

```diff
@@ -64,8 +63,9 @@ sub download{
 
     if( $response->is_success() ){
         my $size = (-s $pathname);
+	$self->{jm}->add("Download_bytes", $size);
```

Comment:
Could add a label here based on the hostname we are downloading from?

Reply:
Riiight.

I did not think about labels, but it's not a huge change to add.

Reply:
Added issue + resolving convo.